mob_test.go

```diff
@@ -982,12 +994,14 @@ func createFileAndCommitIt(t *testing.T, filename string, content string, commit
 	git("commit", "-m", commitMessage)
 }
 
-func createFile(t *testing.T, filename string, content string) {
+func createFile(t *testing.T, filename string, content string) (path string) {
 	d1 := []byte(content)
```

Comment:
renaming this variable to something more expressive would be nice (although it's not part of this PR, but still - just have seen that)

Reply:
> Tested it on OS X. Works fine. Some more comments:
> 
> ```
> ➜  mob git:(mob/keep-micro-commits) mob squash-wip
>  > Make sure the final commit is a manual commit before squashing
> ```
> 
> This should be an ERROR because the user expected something to happen and it did not. I would rephrase that as well:
> 
> ```
> ERROR failed to squash wip commits
> ERROR last commit must be a manual commit
> ERROR create a manual commit with a commit message to fix this:
> 
>      git commit --allow-empty
> ```

Thank you for testing. Also, I like your remarks regarding error and guidance.
Improved those messages, to even show the resulting commits. Really like the sayTodo.
Feels nice now.
Also found a bug in the process. An empty commit broke squash-wip. I fixed that.